tests/example.cpp

```diff
@@ -18,9 +18,11 @@ int main() {
   const int b = dbg(3 * a) + 1;  // [example.cpp:18 (main)] 3 * a = 6 (int)
 
   std::vector<int> numbers{b, 13, 42};
-  dbg(numbers);  // [example.cpp:21 (main)] numbers = {7, 13, 42} (std::vector<int>)
+  dbg(numbers);  // [example.cpp:21 (main)] numbers = {7, 13, 42}
+                 // (std::vector<int>)
```

Comment:
Can we please revert the formatting here (and maybe replace `int` with `int32_t`? both in the output/comment as well as in the code.

Reply:
Similar below.

We should probably user `int32_t` throughout this `example.cpp`?

Also, once this file is updated, can you please copy it over to the README example?